### ISSUE TITLE
Migrate from internal curry1 to standard curryN

### ISSUE THREAD

**Issue by char0n** [char0n/ramda-adjunct#1340]
**Is your feature request related to a problem? Please describe.**

Remove dependency on internal ramda API

**Describe the solution you'd like**
Before
```js
import curry1 from 'ramda/src/internal/_curry1';

const isValidNumber = curry1(either(isInteger, isFloat));
```
After
```js
const { curryN } from 'ramda';

const isValidNumber = curryN(1, either(isInteger, isFloat)));
```

**Describe alternatives you've considered**

Leaving status quo

**Additional context**

The less internal ramda API we use the better.

**Comment by willblake01** [comment]
@charOn Can I have this one please? I am very inexperienced but I believe I have a solution for this. Thanks!

**Comment by char0n** [comment]
@willblake01 sure go ahead. This is mostly manual work, checkout the updated description of the issue with there the `curry1` and `curryN` comes from. Do the work in batches, 3-5 files per Pull Request. And GOOD LUCK!

**Comment by willblake01** [comment]
@charOn Thank you so much! Sorry, this is all new to me. I tried to push to my branch and got the following error:

remote: Permission to char0n/ramda-adjunct.git denied to willblake01.
fatal: unable to access 'https://github.com/char0n/ramda-adjunct/': The requested URL returned error: 403

**Comment by willblake01** [comment]
@char0n I forked the repo and created a PR for the first batch of 5. I'm unable to add you as a reviewer, am I missing something?

**Comment by char0n** [comment]
> I'm unable to add you as a reviewer, am I missing something?

Repo maintainers have to assign people for review, don't worry I'll assign my self.

> I tried to simply clone the repo but I was denied access to push my commit.

@willblake01 you have to fork a repo before starting to work on it. You simply don't have permission to this repo, so forking mechanism exists to be able to clone the repo and start working on it: https://help.github.com/en/github/getting-started-with-github/fork-a-repo

I'll attend to you PR shortly. THank you

**Comment by willblake01** [comment]
@char0n I only see 5 more instances of the internal ramda API but they are not included in the ramda library. I'm unsure how to proceed. Are you needing solutions to removing those dependencies on the internal ramda API?

**Comment by char0n** [comment]
@willblake01 I see that you already cleared the codebase from `curry1`. There are no more occurrences. I'll create separate issues for all other ramda internals with documentation how to proceed and I'll ping you with those (probably tommorow). Thank you for processing this issue! Nicely done. 

The changes you made are alredy released in v2.26.0: https://www.npmjs.com/package/ramda-adjunct/v/2.26.0
